packages/flutter/lib/src/material/switch_list_tile.dart

```diff
@@ -11,14 +11,19 @@ import 'theme_data.dart';
 
 // Examples can assume:
 // void setState(VoidCallback fn) { }
-// bool _lights;
```

Comment:
The first example appears to depend on _lights

Reply:
The original issue here was that `_lights` is initialized in the first example to demonstrate best practices within the same code sample that developers will see in the docs. However, it will get initialized twice in the same doc if we keep it up here, and the dartdoc analyzer wasn't too happy about that. So, I switched the examples to use `isSelected` in the second example to make it work.